floats/floats.go

```diff
@@ -567,6 +568,19 @@ func Norm(s []float64, L float64) float64 {
 	return math.Pow(norm, 1/L)
 }
 
+// ParseWithNA converts the string s to a float64 in v.
```

Comment:
I was thinking about that and I think the shortness should win here. It's almost never the case that the word "missing" is used, but "NA" is very common. I'll wait for other input.